calculate_average_kaufco.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+#
+#  Copyright 2023 The original authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+JAVA_OPTS=""
+java $JAVA_OPTS --class-path target/average-1.0.0-SNAPSHOT.jar dev.morling.onebrc.CalculateAverage_kaufco > java_output.log 2>&1 &
+
+# Capture the PID of the Java process
+JAVA_PID=$!
+
+# Monitor the output file for the desired character
+tail -f java_output.log | {
+    read -r line
+    echo "$line"
+    kill -9 $JAVA_PID
+}
```

Comment:
What's the purpose of this, can it be removed?

Reply:
Ah, ok. I suppose it's unmapping the file which takes a while. See latest (currently pending) entry of @thomaswue for dealing with that in a more graceful way.

Reply:
Found it. Works. Thanks!